docs/user/aws/limits.md

```diff
@@ -5,29 +5,88 @@ You can find a comprehensive list of the default AWS service limits published he
 [AWS Service Limits][service-limits]
 
 Below, we'll identify OpenShift cluster needs and how those impact some of those limits.
+We will also use [aws cli](https://aws.amazon.com/cli/) and [jq](https://stedolan.github.io/jq/) to try and explan
+where limits are set (viewable) and how to show your current usage of resources.
+
+Many commands below are based on [Trusted Advisor](https://aws.amazon.com/premiumsupport/technology/trusted-advisor/)
+and often this can be used to print both the limit and the usage for a service. However to check those limits/usage
+you need to first identify a check_id. These check_id's can be pulled from:
+
+```sh
+$ aws support describe-trusted-advisor-checks --language en | jq -c '.checks[] | {ID: (.id), Name: .name, Category: .category}'
```

Comment:
I think `sh` commands shouldn't have `$`  as we want users to copy and run. I remember @wking mention something on those line.

Reply:
> I think `sh` commands shouldn't have `$`...

Yup.  More discussion on `sh` vs. `console` in #27.

Reply:
```
:%s/```sh/```console/g
```
will fix that; I will push changes to address this.